client/extensions/woocommerce/app/store-stats/controller.js

```diff
@@ -75,24 +83,50 @@ export default function StatsController( context, next ) {
 		} );
 	}
 
-	const asyncComponent =
-		props.type === 'orders' ? (
-			<AsyncLoad
-				/* eslint-disable wpcalypso/jsx-classname-namespace */
-				placeholder={ <StatsPagePlaceholder className="woocommerce" /> }
-				/* eslint-enable wpcalypso/jsx-classname-namespace */
-				require="extensions/woocommerce/app/store-stats"
-				{ ...props }
-			/>
-		) : (
-			<AsyncLoad
-				/* eslint-disable wpcalypso/jsx-classname-namespace */
-				placeholder={ <StatsPagePlaceholder className="woocommerce" /> }
-				/* eslint-enable wpcalypso/jsx-classname-namespace */
-				require="extensions/woocommerce/app/store-stats/listview"
-				{ ...props }
-			/>
-		);
+	let asyncComponent;
+
+	switch ( props.type ) {
+		case 'orders':
+			asyncComponent = (
+				<AsyncLoad
+					/* eslint-disable wpcalypso/jsx-classname-namespace */
+					placeholder={ <StatsPagePlaceholder className="woocommerce" /> }
```

Comment:
Minor - maybe define a placeholder variable so you only have to do the eslint rules once. Looks like it's the same placeholder in all three cases.